### PR TITLE
Building homepage

### PR DESCRIPTION
**Pull request by Huulamnguyen**
**What does this PR do?**
- This PR builds the homepage of the Application

**Description of Task to be completed?**
- The Homepage shows the navbar which includes the "Can cook" logo, a search bar, login, and register
- The Homepage also shows random food jokes and random recipes.

**How should this be manually tested?**
- Clone the repository.
- From terminal, `flask run` to  run the App
- Home page at http://localhost:5000
Screen Shot 2021-03-16 at 2 10 10 PM

### CODE REVIEW

**Comment by Huulamnguyen** [comment]
@alexkayabula Can you please review this PR and let me know should I improve anything?

File: config.py [hunk 1]
```diff
@@ -1,39 +1,39 @@
 """ Setting Up the App Configuration"""
 import os
+from secret import API_KEY
 
 class Config: 
-    SECRET_KEY = os.environ.get('SECRET_KEY', 'RecipeApp123') or os.urandom(16)
+    SECRET_KEY = os.environ.get('SECRET_KEY', 'RecipeApp123!@#') or os.urandom(16)
+    API_KEY = os.environ.get(API_KEY, 'None')
```
Comment (alexkayabula) [inline config.py:7]: This change is not related to setting up homepage.

File: config.py [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
     SQLALCHEMY_TRACK_MODIFICATION = False
     SQLALCHEMY_ECHO = False
     DEBUG_TB_INTERCEPT_REDIRECTS = False
     WTF_CSRF_ENABLED = False
-
     @staticmethod
     def init_app(app):
         pass
-    
+# Development
```
Comment (alexkayabula) [inline config.py:15]: This change is not related to setting up homepage.

File: config.py [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
     TESTING = True
     SQLALCHEMY_DATABASE_URI = os.environ.get('TEST_DATABASE') or 'postgresql:///recipe_test'
-
+# Production
 class ProductionConfig(Config):
     """Configurations for Production Environment."""
     ENV = 'production'
     SQLALCHEMY_DATABASE_URI = os.environ.get('PRODUCTION_DATABASE')
-
+# config object
```
Comment (alexkayabula) [inline config.py:31]: This change is not related to setting up homepage.

File: config.py [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
 config = {
     'development': DevelopmentConfig,
     'testing': TestingConfig,
     'production': ProductionConfig,
     'default': DevelopmentConfig
 } 
-
```
Comment (alexkayabula) [inline config.py:37]: This change is not related to setting up homepage.

File: config.py [hunk 5]
```diff
@@ -1,39 +1,39 @@
...
 
 class Config: 
-    SECRET_KEY = os.environ.get('SECRET_KEY', 'RecipeApp123') or os.urandom(16)
+    SECRET_KEY = os.environ.get('SECRET_KEY', 'RecipeApp123!@#') or os.urandom(16)
+    API_KEY = os.environ.get(API_KEY, 'None')
     SQLALCHEMY_TRACK_MODIFICATION = False
     SQLALCHEMY_ECHO = False
     DEBUG_TB_INTERCEPT_REDIRECTS = False
     WTF_CSRF_ENABLED = False
-
```
Comment (alexkayabula) [inline config.py:10]: This change is not related to setting up homepage.

**Review by alexkayabula — CHANGES_REQUESTED** [review]
